docs/plugins/backup.md

```diff
@@ -46,7 +46,7 @@ plugin(
     b.bucket = 'acquia-cloud-database-test'
     b.files = [
```

Comment:
I need the target filename for the tarball, which I either retrieve separately (as of now, in a hash), or go with some string chopping magic. You can also optionally specify the target permissions.
https://github.com/acquia/moonshot/pull/103/files#diff-6cee918ef52228fbe94d38a04372ff42R9